tests/sources/test_github.py

```diff
@@ -587,11 +587,15 @@ async def test_ping_with_unsuccessful_connection():
 
 
 @pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "scopes",
+    ["", "repo", "manage_runner:org, delete:packages, admin:public_key"],
+)
 @patch("connectors.utils.time_to_sleep_between_retries", Mock(return_value=0))
-async def test_validate_config_with_invalid_token_then_raise():
+async def test_validate_config_with_invalid_token_then_raise(scopes):
```

Comment:
The test case does not tally with the function name. This is supposed to test with a invalid token (the `pytest.raises` also imply the exception message is about invalid token)

Could you have a separate test case like `test_validate_config_with_insufficient_scope`?

Reply:
Sure, added separate test case `test_validate_config_with_insufficient_scope` and validated `UnauthorizedException` for invalid token